proto/batch_plan.proto

```diff
@@ -71,6 +71,17 @@ message FilterNode {
   expr.ExprNode search_condition = 1;
 }
 
+message LogRowSeqScanNode{
+  plan_common.StorageTableDesc table_desc = 1;
+  repeated int32 column_ids = 2;
```

Comment:
Better to comment its meaning more clearly. 

Is it the output column ids of the raw mv table? Or including column id of the op column? 

I think it should be the output column ids of the raw mv table.